deploy/dashboards/grafana-dashboard-pulp.configmap.yaml

```diff
@@ -3071,72 +3282,60 @@ data:
           "type": "timeseries"
         }
       ],
-      "refresh": "30s",
-      "schemaVersion": 39,
+      "preload": false,
+      "refresh": "1m",
+      "schemaVersion": 41,
       "tags": [],
       "templating": {
         "list": [
           {
             "current": {
-              "selected": false,
-              "text": "stage",
-              "value": "s"
+              "text": "p",
+              "value": "p"
             },
-            "hide": 0,
             "includeAll": false,
             "label": "cluster",
-            "multi": false,
             "name": "cluster",
             "options": [
               {
-                "selected": false,
+                "selected": true,
                 "text": "prod",
                 "value": "p"
               },
               {
-                "selected": true,
+                "selected": false,
                 "text": "stage",
                 "value": "s"
               }
             ],
             "query": "prod : p,stage : s",
-            "queryValue": "",
-            "skipUrlSync": false,
             "type": "custom"
           },
           {
             "current": {
-              "selected": false,
-              "text": "crcs02ue1-prometheus",
-              "value": "PDD8BE47D10408F45"
+              "text": "crcp01ue1-prometheus",
+              "value": "PC1EAC84DCBBF0697"
             },
-            "hide": 0,
             "includeAll": false,
-            "multi": false,
             "name": "datasource",
             "options": [],
             "query": "prometheus",
-            "queryValue": "",
             "refresh": 1,
             "regex": "/crc${cluster}.*ue1-prometheus/",
-            "skipUrlSync": false,
             "type": "datasource"
           },
           {
             "current": {
-              "selected": false,
-              "text": "crcs02ue1.pulp-stage",
-              "value": "arn:aws:logs:us-east-1:744086762512:log-group:crcs02ue1.pulp-stage:*"
+              "text": "crcp01ue1.pulp-prod",
+              "value": "arn:aws:logs:us-east-1:744086762512:log-group:crcp01ue1.pulp-prod:*"
```

Comment:
This will set the variable we use in grafana stage to use prod datasource, isn't it?

Reply:
Good question. I didn't touched that exactly. I believe it will always happen when you use stage or prod to generate the changes of dashboard. When using the stage dashboard, someone will have to change manually the settings to use the stage source, or vice versa.